config/database.yml

```diff
@@ -23,7 +23,7 @@ default: &default
 
```

Comment:
Don't use your information about your database in this file or add in .gitconfig

Reply:
thats a common config for every one dont edit it without specific need